libqsopt/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package format="2">
+	<name>libqsopt</name>
+	<version>0.6.10</version>
+	<description>Wrapper for the Qsopt linear programming solver</description>
+
+	<url type="website">http://www.math.uwaterloo.ca/~bico/qsopt/index.html</url>
+	<author email="florian.jordan@ipa.fraunhofer.de">Florian Jordan</author>
+	<maintainer email="richard.bormann@ipa.fraunhofer.de">Richard Bormann</maintainer>
+
+	<license>GPL</license>
```

Comment:
Qsopt doesn't tell a concrete licence, only that it is free for academic use, but they also provide a package similiar to Qsopt (http://www.math.uwaterloo.ca/~bico/qsopt/ex/index.html) that is provided under GPL and because we don't provide the source code, I wasn't sure what license to use and I decided to use this...